client/landing/gutenboarding/components/domain-picker-modal/style.scss

```diff
@@ -0,0 +1,51 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../../mixins.scss';
+
+.domain-picker-modal-overlay {
+	// Absolute positioning allows the modal
+	// to reuse the <body> element's scrollbar.
+	position: absolute;
+
+	// This positions the domain picker modal
+	// right below the gutenboarding header,
+	// keeping the header clickable.
+	top: 64px;
```

Comment:
I think this should use `$gutenboarding-header-height` from `client/landing/gutenboarding/variables.scss`.

Reply:
Thanks!